btr-web/btr-common-components/components/bcros/Widgets/Fee/Index.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="w-[282px] bg-white p-0" data-cy="pay-fees-widget">
+    <BcrosCard>
+      <template #header>
+        <slot name="header">
+          <div
+            class="bg-bcGovColor-header text-white h-bcrosRow font-bold p-3 rounded-t-md"
+            data-cy="pay-fees-widget-title"
+          >
+            {{ $t('widgets.feeSummary.title') }}
+          </div>
+        </slot>
+      </template>
+      <template #default>
+        <slot v-if="hasEmptyFees" name="emptyFees" data-cy="pay-fees-widget-empty-fees" />
+        <slot name="default">
+          <div
+            v-for="fee in fees"
+            :key="fee.uiUuid"
+            class="bg-white p-3 border-gray-300 border-b-[1px] flex"
+          >
+            <span class="font-bold text-sm mr-auto">
+              {{ $t(`widgets.feeSummary.itemLabels.${fee.filingTypeCode}`) }}
+            </span>
+            <span v-if="fee.quantity > 1" class="font-normal text-sm text-gray-400 float-right ml-2">
+              {{ `X${fee.quantity}` }}
+            </span>
+            <span class="font-bold text-sm float-right ml-2 overflow-hidden whitespace-nowrap">
+              {{ fee.total === 0 ? $t('widgets.feeSummary.noFee') : displayCanadianDollars(fee.total) }}
+            </span>
+          </div>
+        </slot>
+      </template>
+      <template #footer>
+        <slot name="footer">
+          <div class="bg-white rounded-b-md shadow-md p-3 items-center flex" data-cy="pay-fees-widget-total">
+            <span class="font-bold text-sm mr-auto">
+              {{ $t('widgets.feeSummary.total') }}
+            </span>
+            <span class="font-normal text-sm text-gray-400 float-right ml-2">
+              {{ $t('currency.cad') }}
+            </span>
+            <span
+              v-if="fees?.length > 0"
+              class="font-bold text-2xl float-right ml-2 overflow-hidden whitespace-nowrap"
+            >
+              ${{ total }}
```

Comment:
I think you are missing the dollar signs here and above?

Reply:
Hah, I was either missing it above, or had it extra here :P
Updated this now. Can we see examples in the gcbrun ?